frontend/app/mission/new-mission.html

```diff
@@ -0,0 +1,158 @@
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="11" ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/team.png" ></div>
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="4"  ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/faces.png" ></div>
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="1"  ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/book.png" ></div>
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="9"  ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/play.png" ></div>
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="2"  ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/builder.png" ></div>
+      </div>  
+    <div class="row">
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="5"  ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/hand.png" ></div>
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="6"  ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/home.png" ></div>
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="7"  ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/man.png"></div>
+    <div class="col-sm-2" >
+      <input type="radio" QI="QI" value="8"  ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/math.png"></div>
+    <div class="col-sm-2" >
+    <input type="radio" QI="QI" value="10" ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/question.png" ></div>
+    <div class="col-sm-2" >
+    <input type="radio" QI="QI" value="12" ><img class="img-circle" src="https://s3-us-west-2.amazonaws.com/itreeware/questBuilder/trees.png"></div>
+  </div>
+</div>
+
+Other Attributes <br>
+<div class="row">
+    <div class="col-sm-2" >
+      <label>Min Grade</label> 
+          <select>
+            <option grade="1">Choose</option>
+            <option grade="1">1</option>
+            <option grade="2">2</option>
+            <option grade="3">3</option>
+            <option grade="4">4</option>
+            <option grade="5">5</option>
+            <option grade="6">6</option>
+            <option grade="7">7</option>
+            <option grade="8">8</option>
+            <option grade="9">9</option>
+            <option grade="10">10</option>
+            <option grade="11">11</option>
+            <option grade="12">12</option>
+          </select>
+      </div>
+    <div class="col-sm-2" >
+      <label>Max Grade</label> 
+           <select>
+            <option grade="1">Choose</option>
```

Comment:
Please send enough detail so I can follow up.  How about a code snippet example of how you would do this? TheresaStrepekOracle Middleware DeveloperMobile | Fusion | AngularJS 


-------- Original Message --------
Subject: Re: [Planet-Lab] Login Styling (#39)
From: Walt Askew <notifications@github.com>
Date: Sat, April 18, 2015 8:00 am
To: freedomgames/Planet-Lab <Planet-Lab@noreply.github.com>
Cc: "Tree Strepek (active)" <tree@iTreeware.com>

In frontend/app/mission/new-mission.html: > +            <option grade="3">3</option>
> +            <option grade="4">4</option>
> +            <option grade="5">5</option>
> +            <option grade="6">6</option>
> +            <option grade="7">7</option>
> +            <option grade="8">8</option>
> +            <option grade="9">9</option>
> +            <option grade="10">10</option>
> +            <option grade="11">11</option>
> +            <option grade="12">12</option>
> +          </select>
> +      </div>
> +    <div class="col-sm-2" >
> +      <label>Max Grade</label> 
> +           <select>
> +            <option grade="1">Choose</option>
 All of these look like they're asking for some templated looping action. —Reply to this email directly or view it on GitHub.